go/test/endtoend/cluster/vtgate_process.go

```diff
@@ -131,7 +132,7 @@ func (vtgate *VtgateProcess) WaitForStatus() bool {
 }
 
 // GetStatusForTabletOfShard function gets status for a specific tablet of a shard in keyspace
-func (vtgate *VtgateProcess) GetStatusForTabletOfShard(name string) bool {
+func (vtgate *VtgateProcess) GetStatusForTabletOfShard(name string, count int) bool {
```

Comment:
what is this count for? it is not obvious.

Reply:
count is "how many endpoints to wait for"
Since there are 2 rdOnly tablets we need to wait for 2(count) endpoints.

Reply:
then this should be `WaitFor...` instead of `GetStatusFor..`

Reply:
GetStatusForTabletOfShard(vtgate_process.go) is called mostly in conjuction with WaitForStatusOfTabletInShard (vtgate_process.go)) where wait for endpoint is mentioned. I will make a update comment in GetStatusForTabletOfShard to label it as : number of endpoints.
Done. Refer [PR46](https://github.com/planetscale/vitess/pull/46)